src/app/auth/account/account.component.html

```diff
@@ -90,6 +90,16 @@ <h2 class="text-start pt-3 px-3 h6 organization">Organizations</h2>
                             datasetsData.datasetTotalCount
                         }}</span>
                     </mat-button-toggle>
+                    <mat-button-toggle
+                        data-test-id="account-flows-tab"
+                        value="flows"
```

Comment:
I agreed, I'll move it to a separate folder higher in the hierarchy. But I will do this later, since there will be many conflicts with existing branches